pkg/machine/qemu/stubber.go

```diff
@@ -1,4 +1,4 @@
-//go:build !darwin
+//go:build linux || freebsd
```

Comment:
see https://github.com/containers/podman/pull/21686
AFAIK the goal is to have qemu build on windows there so we should not make it worse here without any reason.

Reply:
The problem is that the qemu provider does compile on windows && mac because of a purposeful change that stubs out core data types (e.g. pkg/machine/vmconfigs/config_windows.go), so since these files are broken with the existing tags they set off validation / lint errors. This is just a temporary least invasive workaround.